components/EmotionList.js

```diff
@@ -52,30 +119,113 @@ export default function EmotionList({
               />
             </StyledCardList>
           ))}
-        </ul>
+        </StyledUlContainer>
       )}
     </>
   );
 }
 
-const StyledHeadline = styled.h2`
-  text-align: center;
+const StyledDivWrapper = styled.div`
+  background-color: #f6f4f3;
+  display: flex;
+  flex-direction: column;
+  overflow-x: hidden;
+  position: relative;
 `;
 
-const StyledFilterList = styled.li`
-  display: inline-flex;
-  list-style-type: none;
+const StyledWrapper = styled.div`
+  padding: 0 20px;
+  position: relative;
+  max-width: 1000px;
+`;
+
+const StyledIconLeft = styled.div`
+  position: absolute;
+  top: 0;
+  left: -30px;
+  height: 100%;
+  width: 120px;
+  font-size: 1.5rem;
+  color: #313366;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  z-index: 1;
+  background: linear-gradient(90deg, #f6f4f3 40%, transparent);
+  cursor: pointer;
+  user-select: none;
+  -webkit-tap-highlight-color: transparent;
+`;
+
+const StyledTabsBox = styled.ul`
+  display: flex;
+  gap: 12px;
+  list-style: none;
+  overflow-x: auto;
+  overflow-y: hidden;
+  scrollbar-width: none;
+  -webkit-overflow-scrolling: touch;
+  margin-right: 30px;
+
+  &::-webkit-scrollbar {
+    display: none;
+  }
+`;
+
+const StyledTab = styled.li`
+  color: #313366;
+  cursor: pointer;
+  font-size: 1rem;
+  white-space: nowrap;
+  background: #f9f9f9;
+  padding: 8px 10px;
+  border: 1px solid #e0e1f0;
+  border-radius: 8px;
+  transition: background-color 0.3s ease;
+  opacity: 70%;
+  -webkit-tap-highlight-color: transparent;
+
+  ${({ $isSelected, $color }) =>
+    $isSelected &&
+    `
```

Comment:
you can use the function `css` from styled-components here, that helps to write this nested css code.
```suggestion
    css`
```